src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+
+public class Tagging {
+    private Person person;
+    private String taggingOperation;
+    private Tag tag;
+    
+    public Tagging(Person person, String taggingOperation, Tag tag) {
```

Comment:
As reviewed in class, you could choose to use a data type less general than a string.